website/source/docs/guides/windows-guide.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "docs"
+page_title: "Windows Service"
+sidebar_current: "docs-guides-windows-service"
+description: |-
+  For our friends running Consul on Windows, we have good news. By using the _sc_ command either on Powershell or 
```

Comment:
I would probably remove the "For our friends running Consul on Windows, we have good news part" Both from here and down in the actual body of the doc.